docs/data_fields.md

```diff
@@ -0,0 +1,28 @@
+
+The main fields are the following, plus the geographical entity fields explained below:
+
+* date: the date the data point refers to
```

Comment:
without a time offset, the date is ambiguous. Unfortunately , a time offset is not unambiguously computable from a time zone.